submissions/JuliaNovak/html-css-popup/index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
```

Comment:
Please make intendants less for better readability

Reply:
what is "intendant"?

Reply:
Sorry, my bad :) Fixed it

Reply:
thanks! also, talking about the 2nd change (Remove padding on hover at the bottom link), the padding appears only while using Tab. If you simply hover over the bottom link using a mouse, there will be no padding. Could you probably give a hint on what's going on there?

Reply:
If I hover with a mouse it also appears. Take a look at CSS properties.
Also related topic - https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity